target/supervisor/supervisord.conf

```diff
@@ -1,28 +1,31 @@
-; supervisor config file
-
 [unix_http_server]
-file=/dev/shm/supervisor.sock    ; (the path to the socket file)
-chmod=0700                       ; sockef file mode (default 0700)
+file = /dev/shm/supervisor.sock
+chmod = 0700
+chown = nobody:nogroup
+username = docker-mailserver
+password = docker-mailserver-password
```

Comment:
Where do these values originate from?

Reply:
I just came up with them. They could be anything, it actually doesn't matter, because they only exist to make the `CRIT`disappear.